dipy/tracking/benchmarks/bench_streamline.py

```diff
@@ -98,7 +104,7 @@ def bench_length():
 
```

Comment:
Because of windows error, can you replace it by 
`assert_array_almost_equal([length_python(s) for s in DATA["length(DATA["streamlines"]), decimal=12)`

Reply:
It should be fine now. I used `assert_array_almost_equal` without default value for `decimal` as we do in the tests.

Reply:
perfect ! Thanks !